README.md

```diff
@@ -128,6 +127,27 @@ salvator-xs-h3-4x2g --ENABLE_ANDROID yes`.
 
 This will require even more time and space, as Android is quite big.
 
+## Build products
+
+During the build some interim and final products will appear.
```

Comment:
Well, `build.ninja` is not a product, strictly speaking. Let's call it "artifact"

Reply:
Fixed. The sentence is reworded.